testkit/src/main/scala/org/bitcoins/testkit/core/gen/p2p/ControlMessageGenerator.scala

```diff
@@ -32,8 +32,10 @@ object ControlMessageGenerator {
 
   def feeFilterMessage: Gen[FeeFilterMessage] = {
     for {
-      fee <- CurrencyUnitGenerator.feeUnit.suchThat(
-        !_.isInstanceOf[SatoshisPerVirtualByte])
+      fee <- FeeUnitGen.feeUnit.suchThat(feeRate =>
```

Comment:
nit: It's usually a bad idea to use `.suchThat()` as it will cause generators to fail if it's hard to satisfy the predicate (in this case, `!feeRate.isInstanceOf[...] && ...`)